src/main/java/cz/xtf/openshift/OpenShiftUtils.java

```diff
@@ -30,22 +31,39 @@ public static OpenShiftUtil admin(String namespace) {
 
 	public static OpenShiftUtil master() {
 		if(masterUtil == null) {
-			String masterUrl = TestConfiguration.masterUrl();
-			String namespace = TestConfiguration.masterNamespace();
-			String username = TestConfiguration.masterUsername();
-			String password = TestConfiguration.masterPassword();
+		    if(!TestConfiguration.openshiftOnline()) {
+		        String masterUrl = TestConfiguration.masterUrl();
+		        String namespace = TestConfiguration.masterNamespace();
+		        String username = TestConfiguration.masterUsername();
+		        String password = TestConfiguration.masterPassword();
 
-			masterUtil = getUtil(masterUrl, namespace, username, password);
+		        masterUtil = getUtil(masterUrl, namespace, username, password);
+		    } else {
+		        String namespace = TestConfiguration.masterNamespace();
+		        String masterUrl = TestConfiguration.masterUrl();
+		        masterUtil = new OpenShiftUtil(new OpenShiftConfigBuilder()
```

Comment:
There is constructor in OpenShiftUtil that takes token instead username and password. Please use that one.